content/collections/data-tables/en/data-tables-results-and-sorting-logic.md

```diff
@@ -31,11 +31,12 @@ When using multiple metrics, sorting by a particular column displays data for al
 
 ## Limits for .CSV exports
 
-Results exported to .CSV have row limits based on the metric type. If the data table contains multiple metrics of different kinds, the smallest row limit applies. Amplitude Analytics prunes rows exceeding the limit, and they don't appear in the exported .CSV.
```

Comment:
are the csv limits in this article the same as the limits described in this article https://amplitude.com/docs/faq/limits.

This article says "Any metric **with** attribution has a  limit of 20 rows." but this article https://amplitude.com/docs/faq/limits says "The limit is 10 with attribution". So is the limit 10 or 20?